### PR TITLE
this works

### PR DESCRIPTION
**Pull request by orka299**
works

### CODE REVIEW

File: collaboration.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
[the remark this answers is not in the document]
Comment (orka299) [reply collaboration.html:1]: it just works - Tood

[the remark this answers is not in the document]
Comment (berra-web) [reply collaboration.html:1]: hey

[the remark this answers is not in the document]
Comment (JoelMoralesa) [reply collaboration.html:1]: Hej klasskompisar!

**Review by JoelMoralesa — COMMENTED** [review]
Joel

**Pull request merged**